src/test/java/ch/wisv/areafiftylan/TicketRestIntegrationTest.java

```diff
@@ -429,4 +430,48 @@ private User createTeamReturnSingleMember() {
 
         return teamMate;
     }
+
+    @Test
+    public void testGetAllTicketsForTransport_Anon() {
+        //@formatter:off
+        when()
+            .get("/tickets/transport")
+        .then()
+            .statusCode(HttpStatus.SC_FORBIDDEN);
+        //@formatter:on
+    }
+
+    @Test
+    public void testGetAllTicketsForTransport_User() {
+        SessionData login = login("user", userCleartextPassword);
+
+
+        //@formatter:off
+        given()
+            .filter(sessionFilter)
+            .header(login.getCsrfHeader())
+        .when()
+            .get("/tickets/transport")
+        .then()
+            .statusCode(HttpStatus.SC_FORBIDDEN);
+        //@formatter:on
+    }
+
+    @Test
+    public void testGetAllTicketsForTransport_Admin() {
```

Comment:
This test is not sufficient by itself. It needs at least another test where there is a difference between all tickets and transport tickets.